index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mint Signup Page</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <!-- Header -->
+    <header>
+        <div class="top-header">
+            <div class="right question">
+                <p>Already have an intuit account?</p>
+                <p><a href="#">I forgot my User ID or Password</a></p>
+            </div>
+            <div class="right signin">
+                <button type="button">
+                    <img src="./images/lock.png" alt="sign in"> 
+                    <span>Sign In</span>
+                </button>
+            </div>
+        </div>
+        <div class="logo center-align">
+            <img src="./images/logo.png" alt="intuit logo">
+        </div>
+    </header>
+    <!-- Main -->
+    <main>
+        <section>
+            <div class="center-align">
+                <h1>Create an Intuit account</h1>
+                <p>One account for everything intuit, including Mint.</p>
+                <a href="#">Learn more</a>
+            </div>
+
+            <form action="/" method="post">
+                <fieldset>
+                    <label for="email">Email address</label>
+                    <input type="email" name="email" id="email" required>
+                </fieldset>
+                <fieldset>
+                    <label for="number">Phone (recommended)</label>
+                    <input type="tel" name="phone number" id="number">
+                    <p>Standard call, messaging or data rates may apply.</p>
+                </fieldset>
+                <fieldset id="input-password">
+                    <label for="password">Password</label>
+                    <img src="./images/lock_open.png" alt="padlock">
+                    <input type="password" name="password" id="password" required>
+                </fieldset>
+                <button class="form-button" type="button" name="submit">
+                    <img src="./images/lock.png" alt="sign in"> 
+                    <span>Create Account</span>
+                </button>
```

Comment:
- [ ] the button should have a `submit` type, not `button`  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))